lib/processor.js

```diff
@@ -0,0 +1,60 @@
+module.exports = {
+  /**
+   *
+   * @param input the string run regex against
+   * @param filters string | object , if string we check if that regex is excluded from input
+   * @returns object { mergeable: bool, description: string | [string]}
+   */
+  processFilters: function (input, filters) {
```

Comment:
We've been using es6 style for functions. Would be good to keep that standard throughout. 

`processFilters: () => {}`

Is it possible to configure standard to error on this I wonder?

Reply:
done, changed it to `const processFilters = () => {}`